### DISCUSSION TITLE
touch sensor with pybullet

### DISCUSSION THREAD

**Discussion by shivanimall** [bulletphysics/bullet3#3818] [Ideas]
recently read on a paper that pybullet has support for tactile simulator engine, but it is not publicly available, wondering if this will be made available in the future?

**Comment by erwincoumans** [comment]
what paper?

**Reply by shivanimall** [reply]
tacto- https://arxiv.org/pdf/2012.08456.pdf
reading back to this- I think I misread, it says recent tactile sensors simulators based on Pybullet are not open source.

**Reply by AlexanderFabisch** [reply]
here is the repository: https://github.com/facebookresearch/tacto
